test/jit.cpp

```diff
@@ -828,3 +828,12 @@ TEST(JIT, setKernelCacheDirectory) {
   // Reset to the old path
   ASSERT_SUCCESS(af_set_kernel_cache_directory(old_path.c_str(), false));
 }
+
+TEST(JIT, UnitTreeSize) {
+    af::array a = af::constant(1, 10);
+    af::array b = af::constant(2, 10);
+    af::array c = a * b;
+    af::array d = b * c;
+    std::cout << af::sum<double>(c) << std::endl;
+    std::cout << af::sum<double>(d) << std::endl;
+}
```

Comment:
I am going to write a better test if possible or remove this one. Accidentally left this print outs here.

Reply:
To test the bug without any environment change, we need a programmatic way of setting the maximum JIT length (per backend).